tests/unit/amo/sagas/test_collections.js

```diff
@@ -732,6 +733,31 @@ describe(__filename, () => {
       );
     };
 
+    it('dispatches addonRemovedFromCollection after removing an add-on from a collection', async () => {
+      const params = {
+        addonId: 123,
+        filters: { page: 2 },
+        slug: 'some-other-slug',
+        username: 'some-other-user',
+      };
+
+      mockApi
+        .expects('removeAddonFromCollection')
+        .once()
+        .returns(Promise.resolve());
+
+      _removeAddonFromCollection(params);
```

Comment:
This can just be `_removeAddonFromCollection()` since none of the parameters are important for testing the behavior of dispatching the removal action.